app/server/app.ts

```diff
@@ -10,6 +10,9 @@ import * as Paths from './paths';
 const bodyParser = require('body-parser');
 import * as cookieParser from 'cookie-parser';
 const { fileTypes } = require('./utils/mimeTypeWhitelist');
+const CONF = require('config');
```

Comment:
can we please rename const CONF to camelCase to be consistent

Reply:
Changed it to use config which is already there